web/packages/core/src/internal/player/inner.tsx

```diff
@@ -1890,6 +1892,7 @@ export class InnerPlayer {
         }
 
         errorArray.push(this.getPanicData());
+        error = originalError;
 
         // Clears out any existing content (ie play button or canvas) and replaces it with the error screen
         showPanicScreen(this.container, error, errorArray, this.swfUrl);
```

Comment:
I would just use `originalError` here.

Reply:
Can't without changes to the logic of the panic screen.

Reply:
https://github.com/ruffle-rs/ruffle/blob/master/web/packages/core/src/internal/ui/panic.tsx#L196

Reply:
I still don't see the difference between `error = originalError; showPanicScreen(...error..)` and `showPanicScreen(..originalError..)`?

Reply:
Oh right, I got confused.